common/frame.go

```diff
@@ -1538,6 +1538,7 @@ type frameExecutionContext interface {
 	ID() runtime.ExecutionContextID
 }
 
+//nolint:unparam
```

Comment:
I would rather keep this warning on and refactor these functions, also `framePointerActionFn`.

Reply:
Sure, but I didn't want to get into a large refactoring now. So we need to keep this `nolint`, and the `TODO` could be "Get rid of all `nolint`s".